lib/fluent/event.rb

```diff
@@ -14,8 +14,6 @@
 #    limitations under the License.
 #
 
-require 'fluent/engine'
```

Comment:
EventStream classes requires `msgpack_factory`. Please add requirement for `fluent/msgpack_factory` and mixin `Fluent::MessagePackFactory::Mixin`, then use `msgpack_factory` (or `msgpack_packer`) method instead of `Engine.msgpack_factory`.